google/cloud/apigeeregistry/v1/scoring/severity.proto

```diff
@@ -37,6 +37,6 @@ enum Severity {
   WARNING = 2;
 
   // This level indicates that the value of the score is a major red flag and
-  // ideally users should activaly take actions to get to a WARNING or OK level.
+  // ideally users should actively take actions to get to a WARNING or OK level.
   ALERT = 3;
 }
```

Comment:
I try very hard not to touch anything beyond what I'm changing, even though I join git in disliking missing newline-at-eof.